tools/realsense-viewer/auto_updater/update_handler.h

```diff
@@ -0,0 +1,53 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2017 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+namespace rs2
+{
+    class update_handler
+    {
+    public:
+        enum update_policy_type { EXPERIMENTAL, RECOMMENDED, REQUIRED};
+        enum component_part_type  { LIBREALSENSE , VIEWER, DEPTH_QUALITY_TOOL, FIRMWARE};
+
+        update_handler() : update_handler("") {}; // Temp Read From local file - REMOVE!
+        update_handler(const std::string &url) : _dev_info_url(url), _server_versions_vec(), _server_versions_loaded(false) {};
+        ~update_handler() {};
+
+        
+        long long check_for_updates(const component_part_type cp, const update_policy_type up);
+        bool get_download_link(const component_part_type part, const long long version, std::string& dl_link);
+
```

Comment:
need to add more info - get version string, get description, get release notes link, etc...

Reply:
Done